pkg/ebpf/tracer_test.go

```diff
@@ -384,13 +406,24 @@ func TestTCPRemoveEntries(t *testing.T) {
 	_, ok := findConnection(c.LocalAddr(), c.RemoteAddr(), connections)
 	assert.False(t, ok)
 
-	// Assert the TCP map is empty because of the clean up
-	key, nextKey := &ConnTuple{}, &ConnTuple{}
+	// Assert the TCP map does not contain first connection because of the clean up
 	tcpMp, err := tr.getMap(bytecode.TcpStatsMap)
-	assert.Nil(t, err)
-	// This should return false and an error
-	err = tcpMp.NextKey(unsafe.Pointer(key), unsafe.Pointer(nextKey))
-	assert.EqualError(t, err, ebpf.ErrKeyNotExist.Error())
+	require.NoError(t, err)
+
+	key, err := connTupleFromConn(c, 0)
+	require.NoError(t, err)
+	stats := new(TCPStats)
+	err = tcpMp.Lookup(unsafe.Pointer(key), unsafe.Pointer(stats))
+	if !assert.True(t, errors.Is(err, ebpf.ErrKeyNotExist)) {
```

Comment:
Doesn't `EqualError` return a `bool`?

Reply:
Also `Logf` entries are only printed on failure or if `-test.v` is set, so do we just want to do this regardless of `EqualError` returning true or false?

Reply:
> Doesn't EqualError return a bool?
`EqualError` is stricter than `errors.Is`, the latter can unwrap error to find the source error.

> Also Logf entries are only printed on failure or if -test.v is set
It was my intention to only have them printed in those cases.